source/core_sntp_client.c

```diff
@@ -173,6 +173,8 @@ static uint32_t calculateElapsedTimeMs( const SntpTimestamp_t * pCurrentTime,
  * - #SntpSuccess if the context is verified to be initialized.
  * - #SntpErrorBadParameter if the context is NULL.
  * - #SntpErrorContextNotInitialized if the context is validated to be initialized.
+ * - #SntpErrorChangeServer if no more servers are remaining from the configured list to
+ * synchronizing time from.
```

Comment:
The second line should probably be justified with the rest of the bullet point. Also, "synchronizing" -> "synchronize"
```suggestion
 * - #SntpErrorChangeServer if no more servers are remaining from the configured list
 *   from which to synchronize time.
```